### DISCUSSION TITLE
can't find storage with rawPath: /

### DISCUSSION THREAD

**Discussion by AlleNPooh** [AlistGo/alist#1635] [General]
升级到最新版本就出现了这个提示；查了很多地方未果，看大神在有一个地方回复了一下，但是没看明白；

**Comment by HHBK201** [comment]
添加存储资源就好啦~

**Reply by AlleNPooh** [reply]
一直以为是哪里设置的有问题。在document中找，也找不到这个相关的解释。

**Reply by AlleNPooh** [reply]
其实有个关键问题是，我原来的数据都没有了。

**Reply by HHBK201** [reply]
我看前面的issues说是v3与v2不兼容

**Comment by chen8945** [comment]
你真就不看我在上一条给你的回复的信息

**Reply by AlleNPooh** [reply]
抱歉，看了的，方向错了，我一直以为我的数据都没丢。

**Reply by chen8945** [reply]
> 抱歉，看了的，方向错了，我一直以为我的数据都没丢。

你找个工具打开数据库文件，v2挂载盘数据还在里面，只是改了些数据结构，你跟着改一下就行或者你复制出来再添加一下就行
